### ISSUE TITLE
support nested inline macro

### ISSUE THREAD

**Issue by zingdle** [microsoft/vscode-cpptools#10647]
### Feature Request

Currently inline macro feature (https://github.com/microsoft/vscode-cpptools/issues/4183) only support 1 level expand:

```c
#define INNER_MACRO(name)   int name = 0
#define OUTER_MACRO(name)   INNER_MACRO(name)

OUTER_MACRO(x);
```

I have to `inline macro` twice to get `int x = 0;`

It's helpful to expand all nested macro when reading unfamilar code.

**Comment by sean-mcmanus** [comment]
I'm not reproing the issue -- I get `'int x = 0;`. What version of the extension, OS, and compiler are you using?

**Comment by zingdle** [comment]
Looks like I used the outdated extension. Newest extension has solved the problem.

Closing...

**Comment by sean-mcmanus** [comment]
Yeah, it was most likely fixed in 1.14.1 by https://github.com/microsoft/vscode-cpptools/issues/3579
